src/gui/modals/SetupDialog.cpp

```diff
@@ -1240,6 +1252,22 @@ void SetupDialog::resetBufferSize()
 	setBufferSize(DEFAULT_BUFFER_SIZE / BUFFERSIZE_RESOLUTION);
 }
 
+void SetupDialog::updateSampleRates()
+{
+	m_sampleRate->clear();
+	if (m_audioInterfaces->currentText() == "JACK (JACK Audio Connection Kit)")
+	{
+		m_sampleRate->setDisabled(true);
+	}
+	else
+	{
+		// TODO: Dynamically get available sample rates from backend
+		m_sampleRate->addItems({"44100","48000","88200","96000","176400","192000"});
```

Comment:
Wouldn't this add duplicate items when you change audio interface back and forth?

Reply:
Nope, `m_sampleRate->clear()` clears the previous content.